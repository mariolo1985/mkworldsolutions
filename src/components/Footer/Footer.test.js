describe('Footer component', () => {
    describe('Initial Render', () => {
        it('Renders correctly', () => {
            const testBoolean = true;
            expect(testBoolean).toEqual(testBoolean);
        });
    });
});
