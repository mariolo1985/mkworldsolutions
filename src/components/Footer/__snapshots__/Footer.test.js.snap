// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Initial Render Renders correctly 1`] = `null`;
